spec/models/package_manager/nu_get_spec.rb

```diff
@@ -221,4 +222,91 @@ def stub_releases(releases_data = {})
       end
     end
   end
+
+  describe "::fetch_canonical_nuget_name" do
+    subject(:result) do
+      VCR.use_cassette(cassette) { described_class.fetch_canonical_nuget_name(name) }
```

Comment:
Would it be possible to do an example without a cassette, such as using a formatted string/snippet with the minimum amount of data needed? That way, it'll be easier for the next person to quickly identify what it is to look for, vs looking through the code + cassette to figure out what the expected structure is.